### PR TITLE
Added GraphQL/Elixir post

### PR DESCRIPTION
**Pull request by richeyb**
Adding a post on GraphQL and Elixir/Phoenix! This is intended to be part 1 of a series (although I'm not sure how many posts it will be total!)

### CODE REVIEW

File: posts/elixir-graphql-part-one.md [hunk 1]
```diff
@@ -0,0 +1,311 @@
...
+`id`, `inserted_at`, and `updated_at` will all be included as default columns for any Ecto tables we create, so we don't need to worry about how we're going to define those.
+
+For the `event_type`, `payload`, and `message` columns, we're going to use those to track what types of events (unsurprisingly) each event is and message will store a condensed version of the payload. Separately, we have another column called `payload`, whose responsibility is to store the full payload when necessary. This allows the `message` column to stay smaller and easier to search!
+
+#### Getting our database tables running in Ecto
+
+Let's get started by building out this table using Phoenix's built-in generators:
+
+```
+$ mix phx.gen.context Log Events events event_type:string message:string payload:text
```
Comment (binarylogic) [inline posts/elixir-graphql-part-one.md:169]: Any particular reason you're namespacing with `Log` here? It appears the files generated are not consistent. Ex: `events.ex` and then `log/log.ex` / `log/log_test.exs`. I think we should just stick with `LogEvent` (one word).

File: posts/elixir-graphql-part-one.md [hunk 2]
```diff
@@ -0,0 +1,311 @@
...
+
+```
+defmodule Greenfy.Schema do
+  use Absinthe.Schema
+
+  import_types Greenfy.Schema.DataTypes
+
+  query do
+    @desc "Get a list of events"
+    field :events, list_of(:event) do
```
Comment (binarylogic) [inline posts/elixir-graphql-part-one.md:253]: This should probably be `list_of(non_null(:event))`, but I think it's fine for the purposes of this tutorial.

**Review by binarylogic — CHANGES_REQUESTED** [review]
Great work! In general I thought this was a good introductory post.

One thing regarding punctuation, there are a lot of sentences that end with an `!` mark. Could we remove most of them and reserve it for sentences that require it?

Other than that I'm 👍

File: posts/elixir-graphql-part-one.md (hunk 1 above)
Comment (richeyb) [reply posts/elixir-graphql-part-one.md:169]: Log is being used as the Context for the Events schema. events.ex would be used to maintain the specific Events schema reference, whereas Log would be used for the general Log concepts and may not be specific only to Events. I worry that `LogEvent` would narrow the scope of the context a little too much.

File: posts/elixir-graphql-part-one.md (hunk 2 above)
Comment (richeyb) [reply posts/elixir-graphql-part-one.md:253]: Yeah, I think that'll be a nice point to explore in a later post around modifiers to field descriptions!

**Comment by richeyb** [comment]
Made the suggested changes! You were not kidding about the sheer number of exclamation points; not sure how I fell into that.

**Review by binarylogic — APPROVED** [review]

**Pull request merged**